include/nil/crypto3/zk/snark/systems/plonk/placeholder/preprocessor.hpp

```diff
@@ -386,20 +386,26 @@ namespace nil {
                         std::array<polynomial_dfs_type, 2> &q_last_q_blind,

```

Comment:
Can we make these parameters const references, because we don't change them. Please check if they are used outside this function, if they can be moved here, we can remove the reference here, and move the values.

If we cannot move it, please add const, so when I see this function I know it can't be moved.

Reply:
yes we can make them const references.
updated in 528edcf5